### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by roym5**

### CODE REVIEW

**Comment by Hunt-E** [comment]
This is good and it can merge, just one small thing to change.

In the first if statement inside of nextTurn, we need to make sure to implement the method that Abbi is making to check if a player has all card facing up. This way, it continues onto the turn if everyone has at least one facedown card and then reshuffles the deck and gives out hands again if someone has ended the hole.

**Comment by Hunt-E** [comment]
Sorry, I mean we need to change that after Abbi's method is implemented. It is good for now and will need to be changed later.

File: src/csc439teamplatypus/cardgame/golfgame/CLIView.java [hunk 1]
```diff
@@ -164,6 +165,10 @@ public void nextTurn() {
             System.out.println("\n");
             nextTurn();
         }
+        else if(cardsRemaining()){
```
Comment (nflanagan6) [inline src/csc439teamplatypus/cardgame/golfgame/CLIView.java:168]: @roym5 @Hunt-E @abbiland wouldn't this end the game if the players used all the cards in the deck without finishing the hole they were on?

Comment (Hunt-E) [reply src/csc439teamplatypus/cardgame/golfgame/CLIView.java:168]: Yes it would, and we are working on fixing that now.

**Pull request closed without merging**